pipeline_manager.py

```diff
@@ -0,0 +1,40 @@
+""" Manages logic for moving the message through the pipeline. """
+import importlib, logging
+from datetime import datetime
+from message import PipelineHistory, NotificationsMessage
+# Imports for all the pipeline steps
+from pipeline_steps.user_targeting import PipelineUserTargeting
+from pipeline_steps.user_policy import PipelineUserPolicy
+from pipeline_steps.presentation import PipelinePresentation
+from pipeline_steps.delivery import PipelineDelivery
+
+def import_from_string(class_name):
+    # TODO: Remove during refactoring
+    class_lookup = {
+    "PipelineUserTargeting": "pipeline_steps.user_targeting",
+    "PipelineUserPolicy": "pipeline_steps.user_policy",
+    "PipelinePresentation": "pipeline_steps.presentation",
+    "PipelineDelivery": "pipeline_steps.delivery"}
+    module_name = class_lookup[class_name]
+    return getattr(importlib.import_module(module_name), class_name)
+
+class Pipeline(object):
+    def __init__(self):
+        self.pipeline_step_list = [
+        "PipelineComposer",
+        "PipelineUserTargeting",
+        "PipelineUserPolicy",
+        "PipelinePresentation",
+        "PipelineDelivery"
+        ]
+    def pipeline_process(self, message):
```

Comment:
You can rename this method to just "process" - it's brief and concise.  The fact that it's within the Pipeline class implies that it's a pipeline_process.